platform/ios/CHANGELOG.md

```diff
@@ -30,6 +27,9 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Fixed an issue where `-[MGLMapView convertCoordinateBounds:toRectToView:]` would return an empty CGRect if the bounds crossed the antimeridian. ([#12758](https://github.com/mapbox/mapbox-gl-native/pull/12758))
 * Fixed bug where the completion block passed to `-[MGLMapSnapshotter startWithQueue:completionHandler:]` was not being called in all code paths. ([#12355](https://github.com/mapbox/mapbox-gl-native/pull/12355))
 * Fixed several crashes related to telemetry collection. ([#12825](https://github.com/mapbox/mapbox-gl-native/pull/12825))
+* Fixed a cubic-bezier interpolation bug. ([#12812] (https://github.com/mapbox/mapbox-gl-native/issues/12812))
+* Fixed an issue that could cause "allow-overlap" symbols to fade in during pan operations instead of always showing. ([#12683] (https://github.com/mapbox/mapbox-gl-native/issues/12683))
+* Added localization for Arabic settings and accessibility translations. ([#12793](https://github.com/mapbox/mapbox-gl-native/pull/12793/))
```

Comment:
Thanks, I initially decided against adding this blurb in https://github.com/mapbox/mapbox-gl-native/pull/12793#issuecomment-418567708, but it can’t hurt to be explicit about it.